### PR TITLE
Derive the identd ident from the Lurker account, admin-assignable (#643)

### LINKED ISSUE #643: Ident: identd default should be user's lurker account name, not their nick
Not sure how this happened but it seems like self-hosted identd is set to the users nick and not their lurker account name, which sort of defeats the purpose of the identd. If we allow it be customized, it should be something the admin can customize per user, but likely not something user can select themselves.

### PR DESCRIPTION
**Pull request by amiantos**
Closes #643.

## The bug

Standalone derived the identd answer from `network.username || network.nick` — both values the account holder can retype at will. On a multi-user instance that defeats the identd: a member could answer as another member's ident, or churn it to walk away from a ban. Node/hosted was already correct (`lu<accountId>`), which is why the issue reads as "not sure how this happened" — only the standalone branch was wrong.

## What changed

- **Standalone ident = the Lurker account name.** The per-network username and nick no longer feed it at all.
- **Admin-assignable per account** — nullable `users.ident`, `PUT /api/admin/users/:id/ident`, inline editor in admin → users. Members have no route to their own; the whole surface is behind `requireAdmin`.
- **Node edition unchanged** and the override route 409s there: the control plane owns hosted identity, and a cell-local override would break fleet-wide uniqueness.
- **Collisions refused** on assignment, compared against every other account's *effective* ident (most accounts derive theirs from the username, so checking stored overrides alone would miss nearly every real collision). Ident-illegal input is rejected rather than silently reshaped into an identity the admin didn't type.
- **Duplicates surfaced, not faked away.** Usernames are looser than idents (spaces, case, 64 chars), so `bob smith` and `bobsmith` both derive `bobsmith`, and no signup path consults idents. Making the derivation collision-aware would mean mixing in the row id on a clash — which makes an account's ident *move* when an unrelated account signs up, and an ident that moves is worse than one that's ambiguous because bans are keyed on it. So the derivation stays stable and the panel flags duplicates for the operator to settle with an override.
- **UI gated on whether anything answers ident lookups.** identd is opt-in and most self-hosts never enable it; when nothing answers, the whole surface collapses to one line naming the env var.

## Behavior change

A self-host **already running identd** (opt-in, mostly multi-user) will see idents change from the network username/nick to the account name on each connection's next connect. Not backfilled from the old value — a multi-network account has no single value to backfill from, and preserving it would preserve the bug. Operator's call, made deliberately.

## Scope of the guarantee

`network.username` still rides the USER command, so on a network that never completes an ident lookup a member can still appear as `alice!~bob@host`. The leading tilde is IRC's standard "unverified" marker and every IRC client sends a user-chosen USER value, so this is left alone. `shared/ident.ts` documents the real scope: when an identd answers, the ident is assigned and trustworthy; when none does, nothing on that connection is verified anyway.

## Testing

`npm run check` clean, 2856 tests pass. New coverage: ident derivation + the invariant that every derived ident is also a legal admin-typed one; route tests for assign / clear / reject / 409-collision / duplicate-flagging; store tests for the write-vs-refetch reporting.

Two review passes (`/code-review high`) found six defects between them — derived idents that the code's own validator rejected (`-bob`), a clear-the-override path that could be refused forever, a placeholder that misstated what clearing produces, and a successful save that could report as failed. All fixed in the follow-up commits, each with a test that fails without the fix.

**Not QA'd against a live network** — the operator doesn't run identd personally and the hosted cells use their own path, so this rests on the test suite and review rather than manual verification.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: vue_client/src/components/admin-panes/UsersPane.vue [hunk 1]
```diff
@@ -15,45 +15,122 @@
...
           <span v-if="u.role === 'admin'" class="role-tag">admin</span>
           <span v-if="u.isPaused" class="paused-tag">paused</span>
+          <span
+            v-if="adminStore.identdEnabled && u.effectiveIdent"
+            class="ident-tag"
+            :title="
+              u.ident
+                ? 'ident assigned by an admin'
+                : 'ident derived from the account name — networks see this in nick!ident@host'
+            "
```
[the remark this answers is not in the document]
Comment (amiantos) [reply vue_client/src/components/admin-panes/UsersPane.vue:47]: Not a concern, there isn't even an admin pane in node mode.
